src/main/resources/application.yml

```diff
@@ -3,9 +3,61 @@ eureka:
     serviceUrl:
       defaultZone: http://localhost:8761/eureka/
   instance:
-    instanceId: ${spring.application.name}:${random.value}
+    instanceId: ${spring.application.name}
 server:
   port: 8095
 spring:
+  data:
+    authService: auth-service
+  url:
+    authServiceUserValidationPath: /v1/validating-token
   application:
     name: api-gateway
+
+#Routing with zuul
+zuul:
+  routes:
+    auth-token-generation:
+      path: /v1/authenticate
+      serviceId: auth-service
+      stripPrefix: false
+    auth-token-validation:
+      path: /v1/validating-token
+      sensitiveHeaders: Cookie,Set-Cookie
+      serviceId: auth-service
+      stripPrefix: false
+    product:
+      path: /api/products
+      sensitiveHeaders: Cookie,Set-Cookie
+      serviceId: product-service
+      stripPrefix: false
+    productid:
+      path: /api/products/*
+      sensitiveHeaders: Cookie,Set-Cookie
+      serviceId: product-service
+      stripPrefix: false
+    usercreate:
+      path: /v1/users**
+      sensitiveHeaders: Cookie,Set-Cookie
+      serviceId: user-service
+      stripPrefix: false
+    userother:
+      path: /v1/user
+      sensitiveHeaders: Cookie,Set-Cookie
+      serviceId: user-service
+      stripPrefix: false
+    userid:
+      path: /v1/users/*
+      sensitiveHeaders: Cookie,Set-Cookie
+      serviceId: user-service
+      stripPrefix: false
+  SendErrorFilter:
+    error:
+      disable: true
+# Auth service
+hostname:
+  userservice: http://localhost:8080
```

Comment:
we should not be using any hardcoded urls

Reply:
Removed.